packages/emulation/libretro-swanstation/package.mk

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="libretro-swanstation"
+PKG_VERSION="6a8c32bb17df77e512a434f622ece1957f391ad8"
+PKG_SHA256="8ef44b88e057159c3b5fded06824494f4d3f5a5eea11e2e6f930ec42b15c3485"
+PKG_LICENSE="GPL-3.0-or-later"
+PKG_SITE="https://github.com/libretro/swanstation"
+PKG_URL="https://github.com/libretro/swanstation/archive/${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="SwanStation is a Libretro core implementation of DuckStation, which is an emulator of the Sony PlayStation console."
+PKG_TOOLCHAIN="cmake"
+
+PKG_LIBNAME="swanstation_libretro.so"
+PKG_LIBPATH="./${PKG_LIBNAME}"
+PKG_LIBVAR="SWANSTATION_LIB"
+
+if [ "${OPENGL_SUPPORT}" = "yes" ]; then
+  PKG_DEPENDS_TARGET+=" ${OPENGL}"
+fi
+
+if [ "${OPENGLES_SUPPORT}" = "yes" ]; then
```

Comment:
Are you sure OpenGL ES is supported? I don't see it in the addon's GPU Render options, and it's not mentioned in swanstation's readme.

Reply:
I'm not 100% sure, since the OpenGL support in Kodi's Retroplayer is not ready yet, however the package in Lakka does support GLES, so I'd expect that the OpenGL renderer supports both desktop GL and GLES. Usually there is only one OpenGL renderer in libretro cores, not 2 separate for OpenGL and OpenGL ES. For now it is software rendering only anyway.

P.S. I've recently tried to get the work in progress OpenGL support from @lrusak to work, but I've ended up in a dead end. Anyone willing to help with this is welcome (https://github.com/KOPRajs/xbmc/tree/retroplayer-gl-wip).